File: Data Structures & Algorithms/set-zeroes-in-matrix/submission-7.cpp
class Solution {
public:
    void setZeroes(vector<vector<int>>& matrix) {
       int row = matrix.size();
       int col = matrix[0].size();
       unordered_set<int> rows,cols;

       for(int i = 0 ; i < row ; i++) 
         {
            for(int j = 0 ; j < col ; j++) 
            {
                if(matrix[i][j] == 0)
                {
                    rows.insert(i);
                    cols.insert(j);

                }
            }

         }

        for(int i = 0 ; i < row ; i++) 
         {
            for(int j = 0 ; j < col ; j++) 
            {
                if(rows.count(i) || cols.count(j))
                {
                     matrix[i][j] = 0;

                }
            }

         }


    }
};

File: Data Structures & Algorithms/length-of-last-word/submission-7.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        count = 0
        i = len(s) - 1

        while i >= 0 and s[i] == ' ':
            i = i -1

        
        while i >= 0 and s[i] != ' ':
          count = count + 1
          i = i -1  
        
        return count

        